examples/WebRadioDemo/WebRadioDemo.ino

```diff
@@ -0,0 +1,131 @@
+/**
+  A simple stream handler to play web radio stations using ESP8266
+
+  Copyright (C) 2018 Vince Gellár (github.com/vincegellar)
+  Licensed under GNU GPL v3
+  
+  Wiring:
+  --------------------------------
+  | VS1053  | ESP8266 |  Other   |
+  --------------------------------
+  |   SCK   |   D5    |    -     |
+  |   MISO  |   D6    |    -     |
+  |   MOSI  |   D7    |    -     |
+  |   XRST  |    -    |    -     |
+  |   CS    |   D1    |    -     |
+  |   DCS   |   D0    |    -     |
+  |   DREQ  |   D3    |    -     |
+  |   5V    |    -    |   VCC    |
+  |   GND   |    -    |   GND    |
+  --------------------------------
+
+  Dependencies:
+  -VS1053 library by baldram (https://github.com/baldram/ESP_VS1053_Library)
+  -ESP8266Wifi
+
+  To run this example define the platformio.ini as below.
+
+  [env:nodemcuv2]
+  platform = espressif8266
+  board = nodemcuv2
+  framework = arduino
+  build_flags = -D PIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH
+
+  lib_deps =
+    ESP_VS1053_Library
+
+  Instructions:
+  -Build the hardware
+    (please find an additional description and Fritzing's schematic here:
+     https://github.com/vincegellar/Simple-Radio-Node#wiring)
+  -Set the station in this file
+  -Upload the program
+
+  IDE Settings (Tools):
+  -IwIP Variant: v1.4 Higher Bandwidth
```

Comment:
So as I see from the build flag you've chosen, it's "IwIP Variant: v2.0 Higher Bandwidth".
Then this comment is outdated ;-)
But I merge it right now. You can provide a separate pull-request to change this if you like.